masterlist.yaml

```diff
@@ -22544,8 +22850,6 @@ plugins:
         itm: 141
   - name: 'Alternate Start - Live Another Life.esp'
     global_priority: 70
-    tag:
-      - C.Location
```

Comment:
Ah, right. I was sure I had checked and double checked for this. Apparently I should have triple checked too. :)